.eslintrc

```diff
@@ -1,33 +1,17 @@
 {
-  "extends": "airbnb-base/legacy",
-  "parserOptions": {
-    "ecmaVersion": 6,
-    "sourceType": "script"
-  },
-  "env": {
-    "es6": true,
-    "amd": true
-  },
-  "plugins": ["prettier"],
+  "extends": "strateg",
```

Comment:
We should use the `strateg/sitevision` config instead. That way we don't need to add these other rules (might need to modify that config instead tho)